src/borg/repository.py

```diff
@@ -438,7 +461,8 @@ def prepare_txn(self, transaction_id, do_cleanup=True):
     def write_index(self):
         hints = {b'version': 2,
                  b'segments': self.segments,
-                 b'compact': self.compact}
+                 b'compact': self.compact,
+                 b'storage_quota_use': self.storage_quota_use, }
```

Comment:
nitpick: that way of using a trailing comma does not minimize the diff if a new element is added afterwards.

should be rather:
```
{a: 1,
 b: 2,
}
```